upsint/utils.py

```diff
@@ -111,10 +111,27 @@ def get_remote_url(remote):
     return url.decode("utf-8").strip()
 
 
-def prompt_for_pr_content(commit_msgs):
+def assemble_pr_template(commit_msgs: str, project_pr_template: Optional[str]) -> str:
+    """
+    Prepare text for the prompt to fill in details when creating a new PR.
+    """
+    # first line is always the PR title
+    template = "Title of this PR\n\n"
+    # here we just either show the project's template or just a list of commit messages
+    if project_pr_template:
+        template = f"{template}{project_pr_template}\n{commit_msgs}"
```

Comment:
You don't need `PR body:` here? I though it's split via that when submitting.

Reply:
the actual splitting is done via `title, body = pr_content.split("\n", 1)`, so the first line is the title, rest is the body